topo_processor/cli/validate.py

```diff
@@ -3,54 +3,73 @@
 import click
 from linz_logger import LogLevel, get_log, set_level
 
-from topo_processor.file_system.get_fs import is_s3_path
+from topo_processor.stac.data_type import DataType
 from topo_processor.stac.validation import validate_stac
 from topo_processor.util import time_in_ms
+from topo_processor.util.configuration import lds_cache_local_tmp_folder
+from topo_processor.util.files import empty_dir
 
 
 @click.command()
+@click.option(
+    "-l",
+    "--layer",
+    required=False,
+    help="The layer number to validate. Take the last version by default.",
```

Comment:
For this to work outside historical imagery. the layerId will have to be hard coded into the historical imagery loader.

Reply:
I'm actually not sure to understand. If we want to run `validate` against another layer than the `Historical Imagery` one, how do you do that?

Reply:
Historical imagery only has one layer that can be used to actually validate it. no other layer has the same table layout that we are expecting.

in the future we will need two layers to actually validate it too.

Reply:
So at the moment `validate` will by default and only run against `Historical Imagery`?